tests/test_query.py

```diff
@@ -124,7 +124,7 @@ def test_query_clone() -> None:
 
 
 def test_bool_converts_its_init_args_to_queries() -> None:
-    q = query.Bool(must=[{"match": {"f": "value"}}])
+    q = query.Bool(must=[{"match": {"f": "value"}}])  # type: ignore
```

Comment:
I could have used the class here, but the whole point of the test is to make sure the dict is equivalent to the class version, so I ignored the typing errors.